xbmc/guilib/GUIColorButtonControl.cpp

```diff
@@ -35,7 +35,7 @@ CGUIColorButtonControl::CGUIColorButtonControl(int parentID,
   m_colorPosY = 0;
   m_imgColorMask->SetAspectRatio(CAspectRatio::AR_KEEP);
   m_imgColorDisabledMask->SetAspectRatio(CAspectRatio::AR_KEEP);
-  m_imgBoxColor = UTILS::COLOR::NONE;
+  m_imgBoxColor = KODI::GUILIB::GUIINFO::CGUIInfoColor(UTILS::COLOR::NONE);
```

Comment:
Because it's used a few times, you can put `using` namespaces at the top and shorten these IDs.

The general rule is that `using namespace KODI;` can go everywhere. The second namespace should only be used in its component folder, so in xbmc/guilib/ you can do `using namespace GUILIB;`. So at the top, between the includes and the definitions, you would do

```c++
using namespace KODI;
using namespace GUILIB;
```

And then the IDs in this file would be

```c++
GUIINFO::GUIInfoColor(UTILS::COLOR::NONE);
```